homework/vector-of-shared-ptrs/vectorFunctions.cpp

```diff
@@ -0,0 +1,36 @@
+#include "vectorFunctions.hpp"
+
+std::vector<std::shared_ptr<int>> generate(int count){
+    std::vector<std::shared_ptr<int>> vec;
+    vec.reserve(count);
+    for(int i = 0; i < count; i++){
+        vec.push_back(std::make_shared<int>(i));
+        //vec.emplace_back(new int(i));
+    }
+    return vec;
+};
+
+void print(std::vector<std::shared_ptr<int>> vec){
+    for(auto pointer: vec){
+        std::cout << *pointer<<std::endl;
+    }
+}
+
+void add10(std::vector<std::shared_ptr<int>> vec){
+    for(auto pointer: vec){
```

Comment:
You can pass be reference to avoid copy and increment `use_counter`

Add space between operator like `+` `-` `/` `:` `&&` etc..